### DISCUSSION TITLE
[Bug]: Podman doesn't work with external Volumes

### DISCUSSION THREAD

**Discussion by scott-love** [neurodesk/neurodesk.github.io-hugo#437] [Bug Report]
### Version

1.6.0

### What platform are you seeing the problem on?

Local PC

### What happened

I keep my data (projects) on an external hard drive, which worked fine when using Docker. However, when I use Podman and set my additional directory to /Volumes/data/test, which is the external drive, I get an error. Using Docker I do not get this error. Using /Users/slove/test, which is NOT an external drive, everything works fine. 

For the moment I just moved my data but I thought it was worth signalling this potential bug. 

I am using:
Apple M1 Max
OS X 14.6.1
Podman v5.2.2

### How to reproduce

Inside Settings set Engine options to Podman and an additional directory that is an external hard drive path and try to create a new session. 

<img width="643" alt="Screenshot 2024-08-27 at 14 39 04" src="https://github.com/user-attachments/assets/faa9908b-1f4c-402b-aa68-e1951131d898">

**Comment by Vbitz** [comment]
Looking at https://stackoverflow.com/questions/69298356/how-to-mount-a-volume-from-a-local-machine-on-podman it seems that Podman mounts your home directory into the machine automatically but that doesn't seem to happen with the external drive.

Based on https://github.com/containers/podman/discussions/13948 it seems like you might have to fix it on the command line with something like `podman machine init --rootful --now --volume /Volumes/data:/Volumes/data`.

**Comment by scott-love** [comment]
Thanks @Vbitz that helped!
I had to mount both $HOME and the external drive to get it to work:
`podman machine init --rootful --now --volume /Volumes/data:/Volumes/data -v $HOME:$HOME`

**Comment by stebo85** [comment]
@iishiishii - can you replicate this bug and add this fix to the GUI app?

**Comment by stebo85** [comment]
We documented this issue now: https://www.neurodesk.org/docs/getting-started/local/neurodeskapp/#add-a-custom-data-directory
